<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileAttributesDialog</class>
 <widget class="QDialog" name="FileAttributesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>407</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>File attributes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="fileNameLabel">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FileName</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="informationGroupBox">
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="informationFullPathHeadLabel">
        <property name="text">
         <string>Full path :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="informationFullPathLabel">
        <property name="text">
         <string>Full path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="informationMimeHeadLabel">
        <property name="text">
         <string>MIME :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="informationMimeLabel">
        <property name="text">
         <string>MIME</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ownershipGroupBox">
     <property name="title">
      <string>Ownership</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
      <item row="0" column="1">
       <widget class="QLabel" name="ownershipUserLabel">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ownershipGroupHeadLabel">
        <property name="text">
         <string>Group :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ownershipUserHeadLabel">
        <property name="text">
         <string>User :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="ownershipGroupLabel">
        <property name="text">
         <string>Group</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="permissionsGroupBox">
     <property name="title">
      <string>Permissions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QCheckBox" name="permissionsGroupReadCheckBox">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="permissionsGroupExecuteCheckBox">
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="permissionsOtherExecuteCheckBox">
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="permissionsOtherReadCheckBox">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="permissionsOwnerLabel">
        <property name="text">
         <string>Owner</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="permissionsOwnerReadCheckBox">
        <property name="text">
         <string>Read</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="permissionsGroupLabel">
        <property name="text">
         <string>Group</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="permissionsOtherLabel">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="permissionsGroupWriteCheckBox">
        <property name="text">
         <string>Write</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="permissionsOtherWriteCheckBox">
        <property name="text">
         <string>Write</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="permissionsOwnerWriteCheckBox">
        <property name="text">
         <string>Write</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="permissionsOwnerExecuteCheckBox">
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="timestampGroupBox">
     <property name="title">
      <string>Time stamp</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0">
      <item row="0" column="0">
       <widget class="QLabel" name="timestampCreatedHeadLabel">
        <property name="text">
         <string>Created : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="timestampLastModifiedHeadLabel">
        <property name="text">
         <string>Last modified : </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QDateTimeEdit" name="timestampCreatedDateTimeEdit">
        <property name="displayFormat">
         <string>yyyy/MM/dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QDateTimeEdit" name="timestampLastModifiedDateTimeEdit">
        <property name="displayFormat">
         <string>yyyy/MM/dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>permissionsOwnerReadCheckBox</tabstop>
  <tabstop>permissionsOwnerWriteCheckBox</tabstop>
  <tabstop>permissionsOwnerExecuteCheckBox</tabstop>
  <tabstop>permissionsGroupReadCheckBox</tabstop>
  <tabstop>permissionsGroupWriteCheckBox</tabstop>
  <tabstop>permissionsGroupExecuteCheckBox</tabstop>
  <tabstop>permissionsOtherReadCheckBox</tabstop>
  <tabstop>permissionsOtherWriteCheckBox</tabstop>
  <tabstop>permissionsOtherExecuteCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileAttributesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileAttributesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
